base_layer/core/src/consensus/consensus_manager.rs

```diff
@@ -107,15 +107,6 @@ where B: BlockchainBackend
         Ok(())
     }
 
-    /// This returns the difficulty adjustment manager back. This can safely be cloned as the Difficulty adjustment
-    /// manager wraps an ARC in side of it.
-    pub fn get_diff_manager(&self) -> Result<DiffAdjManager<B>, ConsensusManagerError> {
```

Comment:
I removed this method because I couldn't verify that cloning the DiffAdjManager at this stage was safe as claimed. The method is not used so deleted as dead code